spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/services/AbstractEventHandler.java

```diff
@@ -58,7 +57,7 @@ public void start() {
 			.ofType(this.eventType)
 			.cast(this.eventType)
 			.transform(this::handle)
-			.retryWhen(Retry.indefinitely().doBeforeRetry((s) -> this.log.warn("Unexpected error", s.failure())))
+			.onErrorContinue((throwable, o) -> this.log.warn("Unexpected error", throwable))
```

Comment:
`onErrorContinue` _drops_ the error, logs it but will keep the subscription running.